nil/services/stresser/stresser.go

```diff
@@ -239,6 +239,7 @@ func (s *Stresser) Run(ctx context.Context) error {
 			ExportMetrics: true,
 		}
 		nilConfig.RPCPort = 8529
+		nilConfig.EnableDevApi = true
```

Comment:
That's a bit strange to enable it for the single instance mode.

Our purpose is to test how consensus works if some participant is killed. But here we can kill all shards and all participants via single call.